CHANGELOG.md

```diff
@@ -29,6 +29,10 @@ and this project adheres to
 
 ### Fixed
 
+- [#5222](https://github.com/firecracker-microvm/firecracker/pull/5222): Fixed
+  `IovDeque` wraparound logic which prevented it from working properly on
+  systems with non 4K pages.
```

Comment:
Might be more interesting to say how this bug would appear for a customer, instead of the technical detail we fixed. E.g. something like "fixed a bug affecting systems with non 4K pages where network devices stop working after 256 packets were exchanged"?